pkg/testutils/release/releases.go

```diff
@@ -56,6 +56,32 @@ func parseReleases() (map[string]Series, error) {
 	return result, nil
 }
 
+// MajorReleasesBetween returns the number of major releases betweeen
+// the two versions passed. Returns an error when there is no
+// predecessor information for a release in the chain (which should
+// only happen if one of the versions passed is very old).
+func MajorReleasesBetween(v1, v2 *version.Version) (int, error) {
+	older, newer := v1, v2
+	if v1.AtLeast(v2) {
+		older, newer = v2, v1
+	}
+
+	var count int
+	currentSeries := versionSeries(newer)
+
+	for currentSeries != versionSeries(older) {
+		count++
+		seriesData, ok := releaseData[currentSeries]
+		if !ok {
+			return -1, fmt.Errorf("no release data for release series %s", currentSeries)
```

Comment:
`currentSeries` is constructed using `v.Major` and `v.Minor` which are both ints. So worst case it would show `v0.0`, but I added a `:` anyway to make it clear there should be something there.